backend/runners/load_representative_points.py

```diff
@@ -38,14 +41,22 @@ def _get_arguments():
 
 
 def _fake_zip(json_features, fake_zip=FAKE_ZIP_CODE):
-    """Fake ZIP."""
+    """Fake ZIP code."""
     for ft in json_features:
         ft['properties']['zip_code'] = fake_zip
     return json_features
 
 
-def _fake_data(json_features, fake_state):
-    """Fake ZIP, state, county, statefp, countyfp and tractce."""
+def _fake_data(json_features, fake_state, attribute_mapping=ATTRIBUTE_MAPPING):
+    """
+    Fake ZIP code, state, county, statefp, countyfp and tractce.
+
+    If all attributes specified in attribute_mapping are present in the features, these values
+    will be used for the new attributes. Otherwise, the constants at the top of the file are used.
+    """
+    if not json_features:
```

Comment:
What?

Reply:
I use the first element of `json_features` to see if all required attributes are present.